packages/engine/src/simulation/engine.rs

```diff
@@ -184,7 +184,7 @@ impl Engine {
             .instrument(tracing::info_span!("context_sync"))
             .await?;
 
-        // Note: the comment below is mostly invalid until state sync is fixed
+        // Note: the comment below is mostly invalid until state sync is fixed:
```

Comment:
@litvand is state sync fixed? If it's not do we have a task to track this?

Reply:
We haven't checked for a while if it works. We do have a [task](https://app.asana.com/0/1199548034582004/1201644875152939/f).

Reply:
I've brought it up in priority in the backlog by a bit more, thanks for linking.